Kickstarter-iOS/Views/Controllers/RootTabBarViewController.swift

```diff
@@ -211,6 +212,13 @@ extension RootTabBarViewController: UITabBarControllerDelegate {
                                didSelect viewController: UIViewController) {
     self.viewModel.inputs.didSelectIndex(tabBarController.selectedIndex)
   }
+
+  public func tabBarController(_ tabBarController: UITabBarController,
+                               animationControllerForTransitionFrom fromVC: UIViewController,
+                               to toVC: UIViewController) -> UIViewControllerAnimatedTransitioning? {
+
+    return self.transitionCrossDissolve
```

Comment:
I _think_ seeing as there's not real state held in the animator you could just do `return ScrollingTransitionAnimator()` here. Although I'm also not sure if it makes more sense to create it once and just refer to it seeing as it will be used for every tab switch. I think animators are pretty inexpensive to create.